suite-native/feature-flags/src/featureFlagsSlice.ts

```diff
@@ -47,5 +47,26 @@ export const featureFlagsSlice = createSlice({
 export const selectIsFeatureFlagEnabled = (state: FeatureFlagsRootState, key: FeatureFlag) =>
     state.featureFlags[key];
 
+const SEND_COINS_WHITELIST: Readonly<NetworkSymbol[]> = ['btc', 'test', 'regtest'];
+
+export const selectIsNetworkSendFlowEnabled = (
```

Comment:
nitpick: It seems to me it belongs more in the module-send or module-accounts-management, but it can stay here. It is temporary anyway...

Reply:
You are right. It should not be here. 879afffe7b